src/python/dxpy/bindings/dxfile_functions.py

```diff
@@ -123,6 +124,72 @@ def download_dxfile(dxid, filename, chunksize=dxfile.DEFAULT_BUFFER_SIZE, append
                                    show_progress=show_progress, project=project, **kwargs)
 
 
+# Check if a program (wget, curl, etc.) is on the path, and
+# can be called.
```

Comment:
Minor, for consistency: our convention is to add function comments/descriptions in triple double quotes.